governance/default.go

```diff
@@ -665,6 +666,14 @@ func getGovernanceCacheKey(num uint64) common.GovernanceCacheKey {
 
 // Store new governance data on DB. This updates Governance cache too.
 func (g *Governance) WriteGovernance(num uint64, data GovernanceSet, delta GovernanceSet) error {
+	g.idxCacheLock.RLock()
+	idx := make([]uint64, 0, len(g.idxCache))
+	copy(idx, g.idxCache)
+	g.idxCacheLock.RUnlock()
+
+	if len(idx) > 0 && num <= idx[len(idx)-1] {
+		return nil
```

Comment:
is it okay not to return an error??

Reply:
Because this is the normal case if snapshot() is called for old blocks, it is not an error.